src/assets/assets.js

```diff
@@ -0,0 +1,33 @@
+import doc1 from "./doc1.png.png";
+import doc2 from "./doc2.png.png";
+import doc3 from "./doc3.png.png";
+import doc4 from "./doc4.png.png";
+import doc5 from "./doc5.png.png";
+import doc6 from "./doc6.png.png";
+import doc7 from "./doc7.png.png";
+import doc8 from "./doc8.png.png";
+
+export const specialityData = [
+  { speciality: "All Doctors" },
+  { speciality: "General physician" },
+  { speciality: "Gynecologist" },
+  { speciality: "Dermatologist" },
+  { speciality: "Gastroenterologist" },
+  { speciality: "Neurologist" },
+  { speciality: "Cardiologist" },
+  { speciality: "Orthopaedist" },
+  { speciality: "Physiotherapist" },
+  { speciality: "Internist" },
+  { speciality: "Ophthalmologist" },
+];
```

Comment:
Nie może to być po prostu tablica specjalizacji, zamiast obiektów w identycznym kształcie? Chyba że dojdzie tutaj więcej informacji, to olej komentarz:)